Main.as

```diff
@@ -14,6 +14,43 @@ void CopyableItem(const string &in name, int value, bool showValue = true) { Cop
 void CopyableItem(const string &in name, uint value, bool showValue = true) { CopyableItem(name, "" + value, showValue); }
 void CopyableItem(const string &in name, float value, bool showValue = true) { CopyableItem(name, "" + value, showValue); }
 
+enum ListType {
+	Simple,
+	Advanced
+}
+
+string PluginsList(Meta::Plugin@[]@ &in plugins, const ListType listType) {
```

Comment:
No need to use `&in` when you're already passing it as a handle (using `@`). Also, `const ListType` doesn't really need to be const.